src/dataloader.py

```diff
@@ -7,86 +7,34 @@
-        '1.2.276.0.7230010.3.1.4.8323329.11566.1517875233.640521.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.11577.1517875233.694347.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.11584.1517875233.731531.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.12062.1517875237.179186.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.13378.1517875244.961609.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.13415.1517875245.218707.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.13620.1517875246.884737.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.14557.1517875252.690062.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.2083.1517875171.71387.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.2187.1517875171.557615.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.2309.1517875172.75133.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.2563.1517875173.431928.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.2630.1517875173.773726.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.3089.1517875176.36192.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.31801.1517875156.929061.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.32302.1517875159.778024.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.32688.1517875161.809571.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.3321.1517875177.247887.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.3352.1517875177.433385.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.3714.1517875179.128897.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.3791.1517875179.436805.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.4134.1517875181.277174.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.4373.1517875182.554858.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.4468.1517875183.20323.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.4843.1517875185.73985.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.4996.1517875185.888529.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.5087.1517875186.354925.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.5278.1517875187.330082.dcm',
-        '1.2.276.0.7230010.3.1.4.8323329.5543.1517875188.726955.dcm'}
-
-        self.batch_size = batch_size
-
-
-        all_paths = np.array(sorted(glob.glob(im_path)))
-        total_paths = []
-
-        for idx in range(len(all_paths)):
-            if all_paths[idx] not in ignore_files:
-                total_paths.append(all_paths[idx])
-
-        all_paths = np.array(total_paths)
-
-        np.random.seed(816)
-        idxList = np.arange(len(all_paths))  # List of file indices
-        randomIdx = np.random.random(len(all_paths))  # List of random numbers
-        # Random number go from 0 to 1. So anything above
-        # self.train_split is in the validation list.
-        trainIdx = idxList[randomIdx < 0.85]
-
-        validateIdx = idxList[randomIdx >= 0.85]
-
-        if testing:
-            self.im_paths = all_paths[validateIdx]
+        all_paths = np.array(sorted(glob.glob(img_path)))
+        if validation:
```

Comment:
Get rid of if, just pass validation to single line

Reply:
Ahh Gotcha.